scripts/lib/cluster.sh

```diff
@@ -56,8 +56,10 @@ function up-test-cluster() {
 function up-kops-cluster {
     KOPS_S3_BUCKET=kops-cni-test-eks-$AWS_ACCOUNT_ID
     echo "Using $KOPS_S3_BUCKET as kops state store"
-    aws s3api create-bucket --bucket $KOPS_S3_BUCKET --region $AWS_DEFAULT_REGION --create-bucket-configuration LocationConstraint=$AWS_DEFAULT_REGION
-    kops_version="v1.22.3"
+    if ! aws s3api head-bucket --bucket $KOPS_S3_BUCKET; then
+        aws s3api create-bucket --bucket $KOPS_S3_BUCKET --region $AWS_DEFAULT_REGION --create-bucket-configuration LocationConstraint=$AWS_DEFAULT_REGION
+    fi
+    kops_version="v1.25.3"
```

Comment:
This looks like it is ignoring the environment variable passed by the GitHub workflow above. Is that intentional?

Reply:
The version variable here is the binary version that we are using for kops. The environment variable defined in workflow is for defining the kubernetes cluster version we want to create